packages/create-remix/templates/_shared_ts/tsconfig.json

```diff
@@ -11,7 +12,7 @@
     "strict": true,
     "baseUrl": ".",
```

Comment:
Won't this conflict? Which is correct?

Reply:
Ah my bad! I should have checked updated dev branch before submitting the PR. Didn't see `"baseUrl"` got added with PR #482. That works great also. I'll close my PR.